### PR TITLE
Wins works

### LINKED ISSUE #1060: Create a wins form.
### Overview
Create a wins form to replace the current google form following the [figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1209%3A0)
The ultimate goal is to have this form populate a google sheet which team managers can review and approve.

### Action Items
- [x] create an html form called something like wins-form.html
- [x] populate it with questions shown on the figma
- [x] name the columns something that makes sense to you and is inline with the code for the wins page

### Resources/Instructions
[figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1209%3A0)
[medium article about how to make a form submit to google sheet](https://medium.com/@dmccoy/how-to-submit-an-html-form-to-google-sheets-without-google-forms-b833952cc175)

### LINKED ISSUE #1066: Refactor the wins page code to be cleaner
### Overview
I learned from akib about the html template element which can be used to significantly reduce the amount of code needed for the wins page

### Action Items
- [x] rewrite the makeCards function to populate a template rather than all the creating elements and appending children that is going on
- [x] make sure that it continues to work with the see more functionality, including the measuring of the div's height.

### Resources/Instructions
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template

### PR DESCRIPTION
**Pull request by jbubar**
This is a collaborative PR between me and @akibrhast. He redesigned the structure of the wins page, and I put the finishing touches. @akibrhast put most of the work in for this pr. 
I also added some code for restructuring the card so that the quotes show up in the right place

Fixes #1066
closes #1060

### CODE REVIEW

**Comment by jbubar** [comment]
<details>
<summary>desktop view</summary>

screencapture-localhost-4000-wins-2021-02-28-09_59_21


</details>
<details>
<summary>Mobile</summary>

screencapture-localhost-4000-wins-2021-02-28-10_00_13

</details>
<details>
<summary>overlay</summary>

Screen Shot 2021-02-28 at 9 59 51 AM

</details>

File: _sass/components/_wins-page.scss [hunk 1]
```diff
@@ -251,42 +293,58 @@
...
+        height: auto;
+        .wins-see-more-div {
+            position: relative;
+            text-align: right;
+        }
+    }
 }
 
 .wins-card-content > * {
+<<<<<<< HEAD
```
Comment (martham0) [inline _sass/components/_wins-page.scss:312]: Is this supposed to be like this? It looks like it's comparing the same css from the master branch and your own branch

**Review by ItsLaszlo — CHANGES_REQUESTED** [review]
I could not run the branch. It keeps exiting with code 1 after running the command ``docker-compose up`` this seems to be due to some scss error.

image

This might be because of this. I have never seen scss written this way it looks like it's comparing css from 2 different branches.

image

**Comment by jbubar** [comment]
@martham0 Thanks for pointing that out. not sure how that slipped through the cracks. Next time I will be a lot more careful with resolving merge conflicts. 
Does that sort it out?

File: docker-compose.yml [hunk 2]
```diff
@@ -10,4 +10,4 @@ services:
       - 4000:4000
```
Comment (akibrhast) [inline docker-compose.yml]: please remove the docke-compose.yml from this commit, since no changes are made in this file in relation to this pull request.

**Review by akibrhast — COMMENTED** [review]
Everything works as expected! Ready to merge once docker-compose.yml has been removed from the commit. @jbubar

File: _includes/wins-hero.html [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+    <img src="/assets/images/wins-page/wins-trophy.png" alt="image of a person being celebrated and holding a trophy">
     <div>
     <h1>Let's celebrate together!</h1>
-    <a href='https://docs.google.com/forms/d/e/1FAIpQLSdF0MgS-M_Soi77qTp-MNfxMGqq85yqsHwvfHDngHAMyDQ8uQ/viewform'><button class='btn btn-primary btn-xl btn--wins'>Share your wins</button></a>
+    <!-- <a href='https://docs.google.com/forms/d/e/1FAIpQLSdF0MgS-M_Soi77qTp-MNfxMGqq85yqsHwvfHDngHAMyDQ8uQ/viewform'><button class='btn btn-primary btn-xl btn--wins'>Share your wins</button></a> -->
+    <a href='/share-your-wins'><button class='btn btn-primary btn-xl btn--wins'>Share your wins</button></a>
+
+    
     </div>
 </div>
```
Comment (ItsLaszlo) [inline _includes/wins-hero.html:10]: Is this file still necessary? There seems to be another one with this exact code in ``wins/wins-hero.html``

**Review by ItsLaszlo — COMMENTED** [review]
> @martham0 Thanks for pointing that out. not sure how that slipped through the cracks. Next time I will be a lot more careful with resolving merge conflicts.
> Does that sort it out?

Yup! I am able to run it.

**Comment by ItsLaszlo** [comment]
Everything seems to be working as expected! Only a quick note about a possible duplicate file mentioned above. 

This might be nitpicky and maybe a cause for another issue to be created but when using the dropdown filters the cursor change isn't accurate. It changes to ``cursor: pointer`` when hovering over anywhere in the dropdown except for the checkbox. On the checkbox, it turns back into a normal cursor. This can be a bit confusing for the user and implies they can select the category by clicking it rather than clicking the checkbox which isn't the case.

File: _includes/wins-hero.html (hunk 3 above)
Comment (akibrhast) [reply _includes/wins-hero.html:10]: @martham0 Good spotting there!
@jbubar Could you please remove the file martha mentioned as well as the file in the root directory named `wins.html`

**Comment by akibrhast** [comment]
> Everything seems to be working as expected! Only a quick note about a possible duplicate file mentioned above.
> 
> This might be nitpicky and maybe a cause for another issue to be created but when using the dropdown filters the cursor change isn't accurate. It changes to `cursor: pointer` when hovering over anywhere in the dropdown except for the checkbox. On the checkbox, it turns back into a normal cursor. This can be a bit confusing for the user and implies they can select the category by clicking it rather than clicking the checkbox which isn't the case.

@martham0 Could you please open a new ticket with what you mentioned here? The same behavior can also be seen on the projects page as well. I am guessing we could make a ticket that calls for an action to fix it in both places?

**Comment by ItsLaszlo** [comment]
> > Everything seems to be working as expected! Only a quick note about a possible duplicate file mentioned above.
> > This might be nitpicky and maybe a cause for another issue to be created but when using the dropdown filters the cursor change isn't accurate. It changes to `cursor: pointer` when hovering over anywhere in the dropdown except for the checkbox. On the checkbox, it turns back into a normal cursor. This can be a bit confusing for the user and implies they can select the category by clicking it rather than clicking the checkbox which isn't the case.
> 
> @martham0 Could you please open a new ticket with what you mentioned here? The same behavior can also be seen on the projects page as well. I am guessing we could make a ticket that calls for an action to fix it in both places?

Will do! The issue can be found here: #1155

**Comment by akibrhast** [comment]
blalbla
long image
<details>
<summary>this is broken</summary>

image

</details>

**Comment by akibrhast** [comment]
Hey @jbubar If I am not mistaken.. This pull request will also close
#1060 .
Could you confirm that, and if it is case , link it to this pull request

**Comment by jbubar** [comment]
> Hey @jbubar If I am not mistaken.. This pull request will also close
> #1060 .
> Could you confirm that, and if it is case , link it to this pull request

@akibrhast. Right! I did. and I will create a new issue for when it is done #1163

Thanks. and thanks for your patience. been moving around a bunch this week. Next week I will have some more free time. Best of luck this sunday :))

**Review by akibrhast — APPROVED** [review]
Everything looks good and ready for merge. @martham0 Thanks for creating that issue, and feel free to go ahead and merge it once you have had the time to review it yourself :D!

**Review by drubgrubby — COMMENTED** [review]
This is really great work!  I love the new way of structuring complex pages.  

The only thing I notice is a small thing on the layout of the cards: The linkedIn and GitHub images seem to be much closer together than they are on the current site.  

Current site: 
image

This PR: 
image

This may be correct, but I didn't see any discussion of it so I'd like to have @daniellex0 sign off on it before we merge.

**Comment by jbubar** [comment]
> The only thing I notice is a small thing on the layout of the cards: The linkedIn and GitHub images seem to be much closer together than they are on the current site.

good catch!

**Comment by drubgrubby** [comment]
@jbubar - Thanks for making the change to the space between the linkedin and github images.  It looks good to me and ready to merge. 

@martham0 - It seems like this is ready to merge.  If you agree, then please go ahead and squash and merge.

**Comment by ItsLaszlo** [comment]
Everything looks good to me. I'll squash and merge!

**Pull request merged**